hw-model/src/model_emulated.rs

```diff
@@ -54,6 +54,14 @@ pub struct ModelEmulated {
     trace_fn: Option<Box<InstrTracer<'static>>>,
     ready_for_fw: Rc<Cell<bool>>,
     cpu_enabled: Rc<Cell<bool>>,
+
+    image_tag: u64,
+}
+impl Drop for ModelEmulated {
+    fn drop(&mut self) {
+        let bitmap = self.code_coverage_bitmap();
+        let _ = caliptra_coverage::dump_emu_coverage_to_file(self.image_tag, bitmap);
```

Comment:
This will make a mess of /tmp; perhaps we should only do this if an environment variable is set (CPTRA_COVERAGE_PATH=...)?

Reply:
sure, like we do in tracing.

Reply:
Done